Lib/fontTools/varLib/plot.py

```diff
@@ -131,6 +131,60 @@ def plotDocument(doc, fig, **kwargs):
 	plotLocations(locations, fig, names, **kwargs)
 
 
+def _plotModelFromMasters2D(model, masterValues):
+	assert len(model.axisOrder) == 1
+	axis = model.axisOrder[0]
+
+	import numpy as np
+	X = np.arange(-1, 1, .05)
+	Y = []
+
+	for x in X:
+		loc = {axis: x}
+		v = model.interpolateFromMasters(loc, masterValues)
+		Y.append(v)
+
+	from matplotlib import pyplot
+
+	pyplot.plot(X, Y, '-')
+	pyplot.show()
+
+
+def _plotModelFromMasters3D(model, masterValues):
+	assert len(model.axisOrder) == 2
+	axis1, axis2 = model.axisOrder[0], model.axisOrder[1]
+
+	import numpy as np
+	X = np.arange(-1, 1, .05)
+	Y = np.arange(-1, 1, .05)
+	X, Y = np.meshgrid(X, Y)
+	Z = []
+
+	for row_x,row_y in zip(X,Y):
+		z_row = []
+		Z.append(z_row)
+		for x,y in zip(row_x,row_y):
+			loc = {axis1: x, axis2: y}
+			v = model.interpolateFromMasters(loc, masterValues)
+			z_row.append(v)
+	Z = np.array(Z)
+
+	from matplotlib import pyplot
+	fix, ax = pyplot.subplots(subplot_kw={"projection": "3d"})
+
+	surf = ax.plot_surface(X, Y, Z)
+	pyplot.show()
+
+
+def plotModelFromMasters(model, masterValues):
```

Comment:
the new function doesn't seem to be used anywhere, do you plan to hook it to main() via some flag I suppose?

Reply:
> the new function doesn't seem to be used anywhere, do you plan to hook it to main() via some flag I suppose?

I can. For now it's a library function for visualization.

Reply:
since there's no tests, would be nice to have an example of how to use

Reply:
Added command-line for it.